src/manifest.ts

```diff
@@ -40,21 +41,23 @@ export const createManifest = async ({ functions, path }: { functions: FunctionR
 }
 
 const formatFunctionForManifest = ({
+  bundler,
+  displayName,
+  generator,
   mainFile,
   name,
   path,
   runtime,
+  runtimeID,
```

Comment:
Yes this was also annoying me.
`runtime` is `js`, `rs`, `go`
Where `runtimeID` now is the actual runtime`nodejs18.x`

Imho, `runtime` should probably called `language`, but I didn't want to change an existing field. Other solution would be instead of `runtimeID` add a field called `nodeVersion` where we simply add the node version that zisi received and use it in bitballoon/FO to decide on the runtime. Although that would not allow the deployment config API to set a runtime, but maybe if they set a node-version would be okay?

Reply:
I am waiting on this for reviewing the rest of the PRs, as this field it's going to be change it makes the other PRs obsolete or in needs of a change. My two cents: I need to always check in code what these 2 are.

`language` and `runtimeID` seems good for me, as it kind of match the description for [AWS lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).

By [this comment](https://github.com/netlify/pod-compute/issues/64#issuecomment-1490060429), looks like we are going to make a PR in every repository touching runtimes, so maybe the effort of renaming them is not that big?

Reply:
I went for using one field for either language or the runtime name.